inst/extdata/aglu/Muller_crops.csv

```diff
@@ -0,0 +1,21 @@
+# File: Muller_crops.csv,
```

Comment:
I suspect the file's name is supposed to be `Mueller_crops.csv`, but I simply copy & paste from the ODS.